### ISSUE TITLE
ניגוד מינימום בין צבע טקסט לצבע רקע

### ISSUE THREAD

**Issue by noamoss** [hasadna/reportit-user#49]
The  following item does not meet the requirement:

![image](https://user-images.githubusercontent.com/3581741/89738683-38fe9200-da83-11ea-827a-6801cf1dcc57.png)

[Contrast Minimum AA](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)

<div dir="rtl">
יחס ניגוד מינימום בין צבע טקסט לצבע רקע צריך לעמוד על 1:4.5 בטקסט רגיל 1:3  כשהטקסט הוא 14 נקודות מודגש ומעלה או 18 נקודות לא מודגש  ומעלה זה כולל טקסט וקישורים לרבות טקסט או קישורים על גבי תמונות כולל גם ניגוד מינימום כשמוקד מקלדת מגיע לקישור או לכפתור ומתקבל אפקט ויזואלי שונה וכאשר מרחפים עם עכבר על קישור או כפתור וכתוצאה כך מתקבל אפקט ויזואלי שונה.

 ניתן להסתייע  בתוסף לדפדפנים שנקרא [WAVE Web Accessibility Evaluation Tool](https://wave.webaim.org/)  לזהות באמצעות כלים  אוטומטיים ובניהם בעיות יחס ניגוד מינימום ניתן להסתייע בכלי שנקרא [Wave Color Contrast Checker](https://webaim.org/resources/contrastchecker/) לבחירת צבעי ניגוד שעומדים בדרישות התקן

</div>

**Comment by noamoss** [comment]
@akariv added my time estimation, please review/approve/update.
